JDK 13 build fail: Error while generating Javadoc

**My problem:** 
After updating my JDK version from 11 to 13, when I run `mvn clean install` I get the following build error:
```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for MicroProfile Fault Tolerance 3.0-SNAPSHOT:
[INFO] 
[INFO] MicroProfile Fault Tolerance ....................... SUCCESS [  2.008 s]
[INFO] microProfile-fault-tolerance-api ................... FAILURE [  4.633 s]
[INFO] microprofile-fault-tolerance-tck ................... SKIPPED
[INFO] microprofile-fault-tolerance-spec .................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  6.819 s
[INFO] Finished at: 2020-02-13T11:47:20Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.1.1:jar (attach-javadocs) on project microprofile-fault-tolerance-api: MavenReportException: Error while generating Javadoc: 
[ERROR] Exit code: 1 - /Users/josephcass/Documents/libertyGit/microprofile-fault-tolerance/api/src/main/java/org/eclipse/microprofile/faulttolerance/FallbackHandler.java:26: error: heading used out of sequence: <H3>, compared to implicit preceding heading: <H1>
[ERROR]  * <h3>Usage</h3>
[ERROR]    ^
[ERROR] 
[ERROR] Command line was: /Library/Java/JavaVirtualMachines/adoptopenjdk-13-openj9.jdk/Contents/Home/bin/javadoc @options @packages
[ERROR] 
[ERROR] Refer to the generated Javadoc files in '/Users/josephcass/Documents/libertyGit/microprofile-fault-tolerance/api/target/apidocs' dir.
```
<br>

**My environment:**
When running JDK 11 I did not have this issue. I updated to the following OpenJDK version which caused the error:
```
openjdk version "13.0.2" 2020-01-14
OpenJDK Runtime Environment AdoptOpenJDK (build 13.0.2+8)
Eclipse OpenJ9 VM AdoptOpenJDK (build openj9-0.18.0, JRE 13 Mac OS X amd64-64-Bit 20200118_155 (JIT enabled, AOT enabled)
OpenJ9   - 6968c18d7
OMR      - 7a1b0239a
JCL      - 9c5a41bd5f based on jdk-13.0.2+8)
```
<br>

**My solution:** 
I can fix this issue by refactoring all instances of `<h3>` to `<h2>` (There are only 2 instance throughout microprofile-fault-tolerance)

--- a/api/src/main/java/org/eclipse/microprofile/faulttolerance/FallbackHandler.java
+++ b/api/src/main/java/org/eclipse/microprofile/faulttolerance/FallbackHandler.java
@@ -23,7 +23,7 @@ package org.eclipse.microprofile.faulttolerance;
  * The type parameter of the handler instance must be assignable to the return type of the method, 
  * where the {@link Fallback} is specified. The container must ensure this type safety. 
  * Otherwise, {@link IllegalArgumentException} should be thrown. 
- * <h3>Usage</h3>
+ * <h2>Usage</h2>
  * <pre>
  * public class MyService {
  *  &#064;Inject OtherService otherService;

--- a/tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/metrics/util/MetricGetter.java
+++ b/tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/metrics/util/MetricGetter.java
@@ -41,7 +41,7 @@ import org.eclipse.microprofile.metrics.MetricRegistry;
  * for retrieving metrics defined by the faulttolerance spec for a specific
  * class and method name.
  * 
- * <h3>Counters</h3>
+ * <h2>Counters</h2>
  * Helper methods for counter metrics follow a common pattern:
  * <ul>
  * <li>{@code getXxxxTotal()} (e.g. {@link #getRetryCallsSucceededNotRetriedTotal()}) gets the raw value of the counter</li>

